docs/reference/components/sources/apache_metrics.cue

```diff
@@ -111,123 +111,7 @@ components: sources: apache_metrics: {
-			required:    true
-			examples: ["http://localhost:8080/server-status?auto"]
-		}
-		_host: {
-			description: "The hostname of the Apache HTTP server"
-			required:    true
-			examples: [_values.local_host]
-		}
-		access_total: {
-			description:   "The total number of time the Apache server has been accessed."
-			relevant_when: "`ExtendedStatus On`"
-			type:          "counter"
-			tags: {
-				endpoint: _endpoint
-				host:     _host
-			}
-		}
-		connections: {
-			description: "The total number of time the Apache server has been accessed."
-			type:        "gauge"
-			tags: {
-				endpoint: _endpoint
-				host:     _host
-				state: {
-					description: "The state of the connection"
-					required:    true
-					examples: ["closing", "keepalive", "total", "writing"]
-				}
-			}
-		}
-		cpu_load: {
-			description:   "The current CPU of the Apache server."
-			relevant_when: "`ExtendedStatus On`"
-			type:          "gauge"
-			tags: {
-				endpoint: _endpoint
-				host:     _host
-			}
-		}
-		cpu_seconds_total: {
-			description:   "The CPU time of various Apache processes."
-			relevant_when: "`ExtendedStatus On`"
-			type:          "counter"
-			tags: {
-				endpoint: _endpoint
-				host:     _host
-				type: {
-					description: "The state of the connection"
-					required:    true
-					examples: ["children_system", "children_user", "system", "user"]
-				}
-			}
-		}
-		duration_seconds_total: {
-			description:   "The amount of time the Apache server has been running."
-			relevant_when: "`ExtendedStatus On`"
-			type:          "counter"
-			tags: {
-				endpoint: _endpoint
-				host:     _host
-			}
-		}
-		scoreboard: {
-			description: "The amount of times various Apache server tasks have been run."
-			type:        "gauge"
-			tags: {
-				endpoint: _endpoint
-				host:     _host
-				state: {
-					description: "The connect state"
-					required:    true
-					examples: ["closing", "dnslookup", "finishing", "idle_cleanup", "keepalive", "logging", "open", "reading", "sending", "starting", "waiting"]
-				}
-			}
-		}
-		sent_bytes_total: {
-			description:   "The amount of bytes sent by the Apache server."
-			relevant_when: "`ExtendedStatus On`"
-			type:          "counter"
-			tags: {
-				endpoint: _endpoint
-				host:     _host
-			}
-		}
-		uptime_seconds_total: {
-			description: "The amount of time the Apache server has been running."
-			type:        "counter"
-			tags: {
-				endpoint: _endpoint
-				host:     _host
-			}
-		}
-		workers: {
-			description: "Apache worker statuses."
-			type:        "gauge"
-			tags: {
-				endpoint: _endpoint
-				host:     _host
-				state: {
-					description: "The state of the worker"
-					required:    true
-					examples: ["busy", "idle"]
-				}
-			}
-		}
-		up: {
-			description: "If the Apache server is up or not."
-			type:        "gauge"
-			tags: {
-				endpoint: _endpoint
-				host:     _host
-			}
-		}
-	}
+	output: metrics: _apache_metrics
```

Comment:
I'm curious why this was moved out of the file? The metrics defined here are specific to this component so I'd probably leave them here for maintainability.